src/Proto.Actor/Actor.cs

```diff
@@ -11,7 +11,7 @@ namespace Proto
 {
     public delegate Task Receive(IContext context);
 
-    public delegate Task Sender(IContext ctx, PID target, MessageEnvelope envelope);
+    public delegate Task Sender(IOutboundContext ctx, PID target, MessageEnvelope envelope);

```

Comment:
Sender now have access to an IOutboundContext, this is because in the case of a RootContext, there is no full actor context, there is only a partial context.

You only have access to Message, MessageHeader here.